### ISSUE TITLE
Missing accessible keyboard navigation in Accordion

### ISSUE THREAD

**Issue by TGlide** [WailAbou/shadcn-svelte-nodep#9]
[WAI-ARIA Accordion Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/) contains info about the keyboard navigations needed for an accessible Accordion 😄

**Comment by WailAbou** [comment]
Should be fixed with this [PR](https://github.com/WailAbou/shadcn-svelte-nodep/pull/18) and this [PR](https://github.com/WailAbou/shadcn-svelte-nodep/pull/19)
